src/components/TextArea/AutoFocusTextArea.js

```diff
@@ -56,7 +56,7 @@ const styles = StyleSheet.create({
     borderRadius: 8,
     fontFamily: fonts.inter[500],
     fontSize: 14,
-    color: COLORS.black,
-    lineHeight: 24
+    color: colors.black,
+    lineHeight: 21
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `lineHeight` property has been changed from 24 to 21. If this change was intentional and tested across different screen sizes and devices, then it's fine. However, if it wasn't tested or the change was unintentional, it could potentially lead to layout issues.
```diff
-    lineHeight: 21
+    lineHeight: 24
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->